packages/snaps-utils/src/manifest/validation.ts

```diff
@@ -141,7 +141,11 @@ export const SnapIdsStruct = refine(
 
 export type SnapIds = Infer<typeof SnapIdsStruct>;
 
-export const ChainIdsStruct = array(ChainIdStruct);
+export const ChainIdsStruct = size(array(ChainIdStruct), 1, Infinity);
+export const MatchersStruct = object({
+  tlds: optional(size(array(string()), 1, Infinity)),
```

Comment:
Should we refine the strings here to validate the schema?

Reply:
What do you mean by refine the strings? I don't think there's any further string validation I can do here?

Reply:
Oh no haha, I meant that the `.` doesn't have to be placed in the strings themselves, that the assumption is there.

Reply:
Ah, yeah, that makes sense actually. Don't mind me 😅